shell/platform/windows/win32_window.cc

```diff
@@ -0,0 +1,207 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/win32_window.h"
+
+namespace flutter {
+
+Win32Window::Win32Window() {
+  dpi_helper_ = std::make_unique<Win32DpiHelper>();
+
+  // Assume Windows 10 1607 for DPI handling.
```

Comment:
It's not clear to me what the impact is if this assumption is violated. Please flesh out these comments, either here on in a bug describing follow-up work.